src/index.html

```diff
@@ -6,8 +6,276 @@
+          </a>
+        </div>
+        <div class="header__title">
+          <h1 class="header__title-text">miami<br>condo kings</h1>
+        </div>
+      </div>
+      <nav id="nav" class="nav">
+        <div class="nav__info">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#home" class="nav__link">home</a>
+            </li>
+            <li class="nav__item">
+              <a href="#information" class="nav__link">about us</a>
+            </li>
+            <li class="nav__item">
+              <a href="#gallery" class="nav__link">gallery</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contacts" class="nav__link">consultation</a>
+            </li>
+            <li class="nav__item">
+              <a href="#testimonials" class="nav__link">testiminials</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contacts" class="nav__link">contacts</a>
+            </li>
+          </ul>
+          <div class="nav__contact-info">
+            <p>+ 1 234 555-55-55</p>
+            <a href="#" class="nav__contact-link">call to order</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="wrapper">
+        <section class="information" id="information">
+          <div class="information__title">
+            <a href="https://www.google.com/maps/d/viewer?msa=0&mid=1nZP5xRXud-FbmkQ9PGiZNGX_Y24&ll=25.789825859747992%2C-80.19366165000001&z=14">
+              <h2 class="page-title-2">
+                Miami best<br>Real estate <span class="page-blue-text">here</span>
+              </h2>
+            </a>
+          </div>
+          <div class="information__text">
+            <h3 class="page-title-3">
+              Over 50 years experience in love.
+            </h3>
+            <p class="page-text">
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p class="page-text">
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p class="page-text">
+              <strong>We will never be late!<br>Guaranteed secure transaction - 100%</strong>
+            </p>
+          </div>
+        </section>
+        <section class="gallery" id="gallery">
+          <div class="gallery__title">
+            <h2 class="page-title-2">Gallery</h2>
+            <div class="title-decoration"></div>
+          </div>
+          <div class="gallery__content">
+            <div class="gallery__pic-1">
```

Comment:
```suggestion
            <div class="gallery__pic-1 gallery__pic--1">
```
You have a linter error `Tag doesn't have a modifiable class` because there's should be a separate element, and the element should have a modifier. So when you are writing just `gallery__pic--1` it means that there's no element yet, but you are trying to give modifier for unexisting element

You can read more about linter errors here:
https://mate-academy.github.io/fed/bemlint/rules-description.html